### PR TITLE
Change stuff to fit our discord

### PR DESCRIPTION
**Pull request by Natan515**
Marge please

### CODE REVIEW

**Comment by Natan515** [comment]
Ok you can add it now.

File: LICENSE.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-Copyright (c) 2020, Liv <~@l1v.in>
+Copyright (c) 2020
```
Comment (l1ving) [inline LICENSE.md:2]: Why do you think you can just do this?
If you really want to bump the copyright, add `lambda-client` on the second line.

Comment (l1ving) [inline LICENSE.md:2]:
```suggestion
Copyright (c) 2020, Liv <~@l1v.in>
Copyright (c) 2021, lambda-client
```

File: kami-blue-plugin/resources/plugin_info.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "KAMI Blue Plugin",
+  "name": "Lambda-client Plugin",
   "version": "1.1.0",
-  "authors": ["Liv"],
-  "description": "Plugin for the KAMI Blue Discord server",
+  "authors": ["Liv, Constructor"],
```
Comment (l1ving) [inline kami-blue-plugin/resources/plugin_info.json:4]: I don't see Constructor's commits anywhere in this pull?

File: kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/InstallCommand.kt [hunk 3]
```diff
@@ -6,20 +6,20 @@ import org.kamiblue.botkt.utils.normal
...
                 )
             } else {
                 channel.normal(
                     title = "Install",
-                    description = "Download this from <#634549110145286156> or from the [website](https://kamiblue.org/download).\n\nThis is a `1.12.2` [Forge mod](https://files.minecraftforge.net/maven/net/minecraftforge/forge/index_1.12.2.html). You can double click the jar to open the installer.\nIf you use MultiMC or something, you can drag the jar file to your instance's mods folder.\n\nUse `;kami` for more information about different versions."
+                    description = "Download this from <#634549110145286156> or from the [website](https://Lambda-client.com/download).\n\nThis is a `1.12.2` [Forge mod](https://files.minecraftforge.net/maven/net/minecraftforge/forge/index_1.12.2.html). You can double click the jar to open the installer.\nIf you use MultiMC or something, you can drag the jar file to your instance's mods folder.\n\nUse `;lambda` for more information about different versions."
```
Comment (l1ving) [inline kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/InstallCommand.kt:22]: You updated literally none of these

File: kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/SafeCommand.kt [hunk 4]
```diff
@@ -12,15 +12,15 @@ object SafeCommand : BotCommand(
...
                 "Both members have full write access to the organization, with both members having **2fa** protection on their Github Account.\n" +
                 "If a commit is not signed, it is **imperative** that you check the contents of the commit yourself, if you'd like to be sure it is safe (ie a pull request was merged without signed commits).\n" +
                 "If the signature is invalid, do not trust that jar, that means it's possible one of our github accounts was hacked.\n\n" +
-                "Keep in mind, the likelyhood of having a Github account with **2fa** and proper password security being hacked is **extremely low**. Liv uses a 12 word + special characters standard, which is virtually uncrackable in the heat death of the universe, along with Github hashing the password, and the commits being signed by **4096 bit RSA**, with iBuyMountainDew using similar security.")
+                "Keep in mind, the likelyhood of having a Github account with **2fa** and proper password security being hacked is **extremely low**. All of the members of Lambda-client have great paswords and 2fa enabled.")
```
Comment (l1ving) [inline kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/SafeCommand.kt:23]: You can't even spell password correctly, I doubt this is true

File: kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/SafeCommand.kt [hunk 5]
```diff
@@ -12,15 +12,15 @@ object SafeCommand : BotCommand(
...
                 description = "**TL:DR Yes, it is.**\n\n" +
-                "KAMI Blue is open source and publically auditable. You can view the full source code [here](https://github.com/kami-blue/client).\n" +
-                "You can view instructions for building a jar from source [here](https://kamiblue.org/contributing).\n" +
-                "Signed commits are required, with signed releases being published by l1ving. " +
+                "Lambda-client is open source and publically auditable. You can view the full source code [here](https://github.com/Lambda-client/client).\n" +
+                "You can view instructions for building a jar from source [here](https://Lambda-client.com/contributing).\n" +
+                "Signed commits are required, with signed releases being published by Constructor." +
                 "Both members have full write access to the organization, with both members having **2fa** protection on their Github Account.\n" +
                 "If a commit is not signed, it is **imperative** that you check the contents of the commit yourself, if you'd like to be sure it is safe (ie a pull request was merged without signed commits).\n" +
```
Comment (l1ving) [inline kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/SafeCommand.kt:21]: Your own commits aren't even signed, with the exceptions of the ones you do via the web editor??

**Review by l1ving — CHANGES_REQUESTED** [review]

File: LICENSE.md (hunk 1 above)
Comment (huddy987) [reply LICENSE.md:2]: Yeah +1, you cannot remove people from the copywrite.

File: kami-blue-plugin/resources/plugin_info.json (hunk 2 above)
Comment (huddy987) [reply kami-blue-plugin/resources/plugin_info.json:4]: +1, don't add Constructor

File: kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/KamiBluePlugin.kt [hunk 6]
```diff
@@ -10,7 +10,7 @@ import org.kamiblue.botkt.plugins.main.managers.ReactionLoggingManager
 class KamiBluePlugin : Plugin() {
 
     override fun onLoad() {
-        Main.logger.debug("KAMI Blue Plugin loading...\n" +
+        Main.logger.debug("Lambda-client Plugin loading...\n" +
```
Comment (huddy987) [inline kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/KamiBluePlugin.kt:13]: Change all these occurances to just Lambda not Lambda-client please

**Review by huddy987 — CHANGES_REQUESTED** [review]

**Comment by huddy987** [comment]
Just a heads up, since we have not yet decided on what to do about the discord bot yet, this may not get merged until we decide and if we decide to continue using this bot.

**Comment by Natan515** [comment]
K

File: kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/InstallCommand.kt (hunk 3 above)
Comment (Natan515) [reply kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/InstallCommand.kt:22]: Dude u told me to remove any credits for u... I would have wanted to give more but U didn't want that...

File: kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/SafeCommand.kt (hunk 5 above)
Comment (Natan515) [reply kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/SafeCommand.kt:21]: They will be

File: kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/SafeCommand.kt (hunk 4 above)
Comment (Natan515) [reply kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/SafeCommand.kt]: I fixed it, it was just a typing error

File: LICENSE.md (hunk 1 above)
Comment (Natan515) [reply LICENSE.md]: Again @l1ving  I wanted to give u credit but u asked me not to!

**Review by Natan515 — COMMENTED** [review]
Well I have fixed what I have been asked to fix and just to be clear l1ving told me to remove any credit of her... I wanted to give her credit.

**Pull request closed without merging**